test/lib/net/dhcp_test.rb

```diff
@@ -56,6 +56,15 @@ class DhcpTest < ActiveSupport::TestCase
     assert_equal record2, record1
   end
 
+  test "record should be equal if one record has no filename" do
+    record1 = Net::DHCP::Record.new(:mac => "aa:bb:cc:dd:ee:ff",
```

Comment:
Looks like `record1` is the same in all of these examples. Can you make a context `equality` that creates `@record1` in `setup`?